src/components/ToggleCurrencyDropdown.tsx

```diff
@@ -6,14 +6,18 @@ import { useBalance } from '@/hooks/boardwalk/useBalance';
 import { customDrawerTheme } from '@/themes/drawerTheme';
 import { formatCents, formatSats } from '@/utils/formatting';
 import { ChevronDownIcon } from '@heroicons/react/20/solid';
+import Tooltip from './utility/Toolttip';
 import RadioButton from './buttons/RadioButton';
 
 const ToggleCurrencyDrawer = () => {
-   const { activeUnit, setActiveUnit } = useCashuContext();
+   const { activeUnit, setActiveUnit, defaultWallets } = useCashuContext();
    const { satBalance, usdBalance, satBalanceInUsd } = useBalance();
    const [isOpen, setIsOpen] = useState(false);
 
    const handleToggle = (unit: Currency) => {
+      if (!defaultWallets.has(unit)) {
```

Comment:
is this even needed if toggle is disabled when it doesn't have it?

Reply:
Its not needed, more of just a sanity check. I preferred to be redundant because there were some bugs showing up when you could toggle currencies even if you didn't have a wallet with that currency

Reply:
would it then make more sense to have that "business logic" inside the `setActiveUnit`? at first sight it seems like a leak of business logic into UI code but no need to act on this now, I am just thinking out loud. 

I noticed that we have some react components that have a bunch of logic that is not strictly UI related and imo we should try to avoid that in the future.

Reply:
ahh yea that makes sense

Reply:
resolving now